cabal.project

```diff
@@ -1,4 +1,4 @@
-packages: common, stub, client, dwarfadt, dwarf, ghc-vis, test, dyepack-test
+packages: common, stub, client, dwarfadt, dwarf, ghc-vis, test, dyepack-test, test, /home/sven/src/head.hackage/packages/io-streams-1.5.1.0
```

Comment:
Getting things to build is one of my least favourite activities :cry: But I managed to figure out how to get this to build. I think you can get rid of this change and instead copy the head.hackage [constraints](https://ghc.gitlab.haskell.org/head.hackage/cabal.constraints) (at least the one for `io-streams`) into this `cabal.project` file (this is described on the [head.hackage page](https://ghc.gitlab.haskell.org/head.hackage/))

Reply:
Ups, Sorry!

`/home/sven/src/head.hackage/packages/io-streams-1.5.1.0` is a left over from the time when I was waiting for my `io-streams` patch being merged to `head.hackage`. (https://gitlab.haskell.org/ghc/head.hackage/-/merge_requests/111)

Reply:
Ah great, so the head.hackage patch is already merged. Still though, when I did `cabal new-build` it selected the wrong is-streams version. Perhaps adding the constraints is not a bad idea (though maybe a separate PR).

Reply:
See here for more build instructions https://github.com/bgamari/ghc-debug/issues/42#issuecomment-693359269